dll/win32/shell32/wine/pidl.c

```diff
@@ -1113,15 +1113,6 @@ LPITEMIDLIST WINAPI SHSimpleIDListFromPathA(LPCSTR lpszPath)
         wPath = HeapAlloc(GetProcessHeap(), 0, len * sizeof(WCHAR));
         MultiByteToWideChar(CP_ACP, 0, lpszPath, -1, wPath, len);
     }
-#ifdef __REACTOS__
-    // FIXME: Needs folder attribute
-    if (PathFileExistsW(wPath))
```

Comment:
This might make the ROS shell look more incorrect in some places but it is 100% correct to remove this hack.

Reply:
Agree.